app/ui/src/app/integration/edit-page/action-configure/action-configure.component.html

```diff
@@ -28,84 +28,88 @@
-                <li>
-                  <a [routerLink]="['/integrations']">Integrations</a>
-                </li>
-                <li>
-                  <a *ngIf="flowPageService.integrationName"
-                     [routerLink]="['/integrations', currentFlowService.integration.id]">{{ flowPageService.integrationName }}</a>
-                  <ng-container *ngIf="!flowPageService.integrationName">New Integration</ng-container>
-                </li>
-                <li class="active">Configure {{ action?.name }}</li>
-              </ol>
-            </div>
-            <div class="toolbar-pf-action-right">
+      <div class="container-fluid">
+        <div class="row toolbar-pf">
+          <div class="col-sm-12">
+            <div class="toolbar-pf-actions">
+              <div class="inline-block">
+                <ol class="breadcrumb">
+                  <li>
+                    <a [routerLink]="['/']">Home</a>
+                  </li>
+                  <li>
+                    <a [routerLink]="['/integrations']">Integrations</a>
+                  </li>
+                  <li>
+                    <a *ngIf="flowPageService.integrationName"
+                      [routerLink]="['/integrations', currentFlowService.integration.id]">{{ flowPageService.integrationName }}</a>
+                    <ng-container *ngIf="!flowPageService.integrationName">New Integration</ng-container>
+                  </li>
+                  <li class="active">Configure {{ action?.name }}</li>
+                </ol>
+              </div>
+              <div class="toolbar-pf-action-right">
+              </div>
             </div>
           </div>
         </div>
       </div>
     </div>
-    <div class="main-panel-body wizard-pf-body clearfix">
-      <div class="title">
-        <h1>{{ step?.connection?.name }}</h1>
-        <h3>{{ action?.name }}</h3>
-      </div>
-      <syndesis-loading [loading]="loading">
-        <ng-container *ngIf="error">
-          <div class="row row-cards-pf">
-            <div class="card-pf">
-              <div class="card-pf-body">
-                <div class="row">
-                  <div class="col-xs-12">
-                    <p [class]="error.class">
-                      <span [class]="error.icon"></span>
-                      {{ error.message }}
-                    </p>
+    <div class="wizard-pf-body syn-scrollable--body">
+      <div class="container-fluid">
+        <div class="title">
+          <h1>{{ step?.connection?.name }}</h1>
+          <h3>{{ action?.name }}</h3>
+        </div>
+        <syndesis-loading [loading]="loading">
+          <ng-container *ngIf="error">
+            <div class="row row-cards-pf">
+              <div class="card-pf">
+                <div class="card-pf-body">
+                  <div class="row">
+                    <div class="col-xs-12">
+                      <p [class]="error.class">
+                        <span [class]="error.icon"></span>
```

Comment:
I strongly encourage you to use `[ngClass]` instad of `[class]`.

Reply:
I was going to address that #1612 but took care of it here instead.